app/controllers/analytics_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AnalyticsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :confirm_two_factor_authenticated
+
+  def create
+    unless analytics_saved?
```

Comment:
If we're going to feature flag things, we should do that here as well. Never trust the client to honor a feature flag.

Reply:
We were more concerned with the UI and a possible popup or something.